hudi-common/src/main/java/org/apache/hudi/common/table/read/HoodieFileGroupReader.java

```diff
@@ -319,9 +319,6 @@ public void close() throws IOException {
     if (recordBuffer != null) {
       recordBuffer.close();
     }
-    if (readerContext != null) {
-      readerContext.close();
```

Comment:
what's the problem here with the close?

Reply:
The reader context doesn't have anything to close so there is no need for this line currently and no need to make the class implement closeable

Reply:
you mean the schema cache is small?

Reply:
No, the schema cache does not actually have anything that requires closing. The cache object is only referenced from the context that creates it. Once that context is no longer referenced, the cache is eligible for GC.